jlm/llvm/ir/operators/IntegerOperations.hpp

```diff
@@ -0,0 +1,472 @@
+/*
+ * Copyright 2025 Nico Reißmann <nico.reissmann@gmail.com>
+ * See COPYING for terms of redistribution.
+ */
+
+#ifndef JLM_LLVM_IR_OPERATORS_INTEGEROPERATIONS_HPP
+#define JLM_LLVM_IR_OPERATORS_INTEGEROPERATIONS_HPP
+
+#include <jlm/rvsdg/binary.hpp>
+#include <jlm/rvsdg/bitstring/type.hpp>
+
+namespace jlm::llvm
+{
+
+/**
+ * This operation is equivalent to LLVM's 'add' instruction.
+ */
```

Comment:
FIXME: Update the documentation as all the operations are only equivalent to the integer part of the instructions.